torch/_subclasses/fake_tensor.py

```diff
@@ -2316,11 +2316,28 @@ def _dispatch_impl(
         converter = self.fake_tensor_converter
 
         is_lift_func = func in self.lift_fns
+
+        # If we are trying to avoid device init, then we need to avoid constant
+        # prop on constant tensors for ops that change devices.
+        avoiding_device_init = False
+        if self.avoid_device_init:
+            if (
+                func == torch.ops.aten._to_copy.default
+                and "device" in kwargs
+                and kwargs["device"].type != "cpu"
```

Comment:
Shouldn't this be checking like, if device == "cuda" then check if torch.cuda.is_available()?